### DISCUSSION TITLE
Welcome to bootstrap-select Discussions!

### DISCUSSION THREAD

**Discussion by caseyjhol** [snapappointments/bootstrap-select#2552] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Asef00** [comment]
A playground on this, is really needed, For people like me, looking for the best solution to their needs.

**Comment by markofla** [comment]
I tried to put two selects in one form, both selects with class=selectpicker. Only the first select works. Nothing happens if I submit my form using the second select. What am I doing wrong?
